deps/mbedtls.mk

```diff
@@ -24,7 +24,7 @@ $(SRCCACHE)/$(MBEDTLS_SRC)/source-extracted: $(SRCCACHE)/$(MBEDTLS_SRC).tar.gz
 	mkdir -p $(dir $@) && \
 	$(TAR) -C $(dir $@) --strip-components 1 -xf $<
 	# Force-enable MD4
-	sed "s|//#define MBEDTLS_MD4_C|#define MBEDTLS_MD4_C|" -i $(SRCCACHE)/$(MBEDTLS_SRC)/include/mbedtls/config.h
+	sed -i"" -e "s|//#define MBEDTLS_MD4_C|#define MBEDTLS_MD4_C|" $(SRCCACHE)/$(MBEDTLS_SRC)/include/mbedtls/config.h
```

Comment:
unfortunately, `-i""` is exactly the same as `-i`, and only works with gsed. Unless you're intentionally naming the backup file `-e`, which perhaps is somewhat clever?

Reply:
Definitely wasn't trying to be clever and just being too hasty. I'll just specify an explicit extension